conference-finder/src/ConferenceItem.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const ConferenceItem = ({ conference }) => {
+  const { name, topic, dateTime, city } = conference;
+  return (
+    <li>
+      <div className="card">
+        <h3>{name}</h3>
+        <p>{topic}</p>
+        <p>{dateTime}</p>
```

Comment:
maybe worth converting this date into a readable format ? by using ` new Date(<dateTime>)` and then taking what you need

Reply:
Have just done this, definitely an improvement!